app/views/PatientsView/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { AgGridReact } from '@ag-grid-community/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { RowClickedEvent } from '@ag-grid-community/core';
+
+import api from '@/services/api';
+import useGrid from '@/hooks/useGrid';
+import withLoading, { WithLoadingInjectedProps } from '@/hoc/WithLoading';
+import snackbar from '@/services/SnackbarUtils';
+import columnDefs from '@/components/ReportsTable/columnDefs.js';
+
+import '../ReportsView/index.scss';
+
+type PatientsViewProps = WithLoadingInjectedProps;
+
+const PatientsView = ({
+  isLoading,
+  setIsLoading,
+}: PatientsViewProps): JSX.Element => {
+  const { patientId } = useParams<{ patientId: string }>();
+  const history = useHistory();
+  const [rowData, setRowData] = useState();
+  const { gridApi, colApi, onGridReady } = useGrid();
+
+  useEffect(() => {
+    if (patientId) {
+      const getData = async () => {
+        try {
+          const { reports } = await api.get(`/reports?searchText=${patientId}`).request();
+          setRowData(reports.map((report) => {
+            const [analyst] = report.users
+              .filter((u) => u.role === 'analyst' && !u.deletedAt)
+              .map((u) => u.user);
+
+            if (!report.patientInformation) {
+              report.patientInformation = {};
+            }
+
+            return {
+              patientID: report.patientId,
+              analysisBiopsy: report.biopsyName,
+              reportType: report.type === 'genomic' ? 'Genomic' : 'Targeted Gene',
+              state: report.state,
+              caseType: report.patientInformation.caseType,
+              project: report.projects.map((project) => project.name).sort().join(', '),
+              physician: report.patientInformation.physician,
+              analyst: analyst ? `${analyst.firstName} ${analyst.lastName}` : null,
+              reportIdent: report.ident,
+              tumourType: report.patientInformation.diagnosis,
+              date: report.createdAt,
```

Comment:
I think there are 2 missing columns here: Bioinformatician and Reviewer

Reply:
good catch, just illustrates how outdated this component was lol

Reply:
[4548417](https://github.com/bcgsc/pori_ipr_client/pull/495/commits/4548417c7d667f901d6c78c8553a1810ea04af1a)